events/events_test.go

```diff
@@ -97,4 +97,55 @@ func TestV2Put(t *testing.T) {
 			}
 		}()
 	}
+
+	// PutRaw
+	failed = 0
```

Comment:
changed this to `atomic.StoreInt64(&failed, 0)`.

I really am just trying to avoid the repeat of the setup part (the const declarations and the initialization of the mocked queue). the memory model here is that each "half" of the test already have `wg.Wait()` to make sure that there won't be concurrency "leak" to the other "half".